ui/widgets/nuxeo-card.js

```diff
@@ -93,10 +93,14 @@ import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
 
           .header .heading {
             @apply --layout-flex;
-            font-size: 1rem;
-            font-weight: 700;
-            letter-spacing: 0.04em;
-            text-transform: uppercase;
+            font-weight: 400;
```

Comment:
Don thing this `.header .heading` block is being used anymore. Is this intended?